packages/data-models/src/language.js

```diff
@@ -9,7 +9,8 @@ export const Lang = {
   PERSIAN: 'per',
   GEEZ: 'gez',
   CHINESE: 'zho',
-  SYRIAC: 'syr'
+  SYRIAC: 'syr',
+  ENGLISH: 'eng'
```

Comment:
I don't actually know if we should normalize 'en' to 'eng'  Was this added for the definition languages? if so, then see other comment on this topic.

Reply:
I agree it may be not a clear cut. I think we can discuss it in the corresponding issue https://github.com/alpheios-project/alpheios-core/issues/597 as this might get nuanced.